include/ccf/endpoint_registry.h

```diff
@@ -39,6 +39,25 @@ namespace ccf::endpoints
     PathTemplateSpec spec;
   };
 
+  struct RequestCompletedEvent
+  {
+    std::string method = "";
+    // This contains the path template against which the request matched. For
+    // instance `/user/{user_id}` rather than `/user/Bob`. This should be safe
+    // to log, though doing so still reveals (to anyone with stdout access)
+    // exactly which endpoints were executed and when.
```

Comment:
This may be particularly questionable in situations where the DynamicJSEndpointsRegistry is configured to use a private table, and the endpoint names and definitions are confidential. It is up the application code to be consistent here though.

Reply:
Agreed, this is a generality but not globally true - we don't log it, we just try to tell you how you might want to log it.